### PR TITLE
OSDOCS-2532: Add overview page for a support book

### PR DESCRIPTION
**Pull request by xenolinux**
- [Jira issue](https://issues.redhat.com/browse/OSDOCS-2532)
- Preview: [link](https://deploy-preview-37766--osdocs.netlify.app/openshift-enterprise/latest/support/index.html)
- OCP version: 4.6+
- QA contacts: 
Storage: @duanwei33 kindly review the bullet point `Storage issues`
Monitoring: @lihongyan1 kindly review the bullet point `Monitoring issues` and `Remote heath monitoring issues`
Network: @zhaozhanqi kindly review the bullet point `Network issues` and `A network trace`
Logging: @IshwarKanse kindly review the bullet points `Bootstrap node journal logs`, `Cluster node journal logs` - LGTM'd
Installation: @pamoedom kindly review the bullet point `installation issues`; also `Gather data about your cluster`
Crio and Node: @sunilcio kindly review the bullet points  `Node issues` and `Crio issues`

### CODE REVIEW

File: _topic_map.yml [hunk 1]
```diff
@@ -1744,7 +1746,7 @@ Name: Nodes
 Dir: nodes
 Distros: openshift-enterprise,openshift-origin
 Topics:
-- Name: Overview of nodes 
+- Name: Overview of nodes
```
Comment (bmcelvee) [inline _topic_map.yml:1749]: This is a test comment for peer review.

File: _topic_map.yml [hunk 2]
```diff
@@ -501,6 +501,8 @@ Name: Support
 Dir: support
 Distros: openshift-enterprise,openshift-online,openshift-dedicated,openshift-origin
 Topics:
+- Name: Support overview
+  File: index
```
Comment (adellape) [inline _topic_map.yml:505]: Test

**Comment by adellape** [comment]
Going to try closing and re-opening this PR to see if it helps clear up the "Line must be part of the diff" review comment error we're getting...

**Comment by IshwarKanse** [comment]
@xenolinux The bullet points Bootstrap node journal logs, Cluster node journal logs looks good to me. Similar to the different component issues sections that you hav added like the Monitoring issues, can you add a section for Logging issues? You can refer this doc link for the same https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-must-gather.html

**Comment by xenolinux** [comment]
> @xenolinux The bullet points Bootstrap node journal logs, Cluster node journal logs looks good to me. Similar to the different component issues sections that you hav added like the Monitoring issues, can you add a section for Logging issues? You can refer this doc link for the same https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-must-gather.html

@IshwarKanse I'm afraid that'd be redundant information (fluff) from the documentation perspective since the [topic](https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-must-gather.html) is covering the must-gather tool usage details. However, we already have `The must-gather tool` bullet point under the `Gather data about your cluster` section. I will do one thing, I will add info about logging from this [topic](https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-must-gather.html) into the existing bullet point so that we cover the logging information.

**Comment by xenolinux** [comment]
> > @xenolinux The bullet points Bootstrap node journal logs, Cluster node journal logs looks good to me. Similar to the different component issues sections that you hav added like the Monitoring issues, can you add a section for Logging issues? You can refer this doc link for the same https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-must-gather.html
> 
> @IshwarKanse I'm afraid that'd be redundant information (fluff) from the documentation perspective since the [topic](https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-must-gather.html) is covering the must-gather tool usage details. However, we already have `The must-gather tool` bullet point under the `Gather data about your cluster` section. I will do one thing, I will add info about logging from this [topic](https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-must-gather.html) into the existing bullet point so that we cover the logging information.

@IshwarKanse I added the information to the `The must-gather tool` bullet point. Let me know if there are any suggestions!

**Comment by IshwarKanse** [comment]
@xenolinux The new bullet point looks more Logging related than the general oc adm must-gather command. 

The oc adm must-gather command is a bit different for Logging as it uses the cluster-logging-operator image for collecting OpenShift Logging related data.
$ oc adm must-gather --image=$(oc -n openshift-logging get deployment.apps/cluster-logging-operator -o jsonpath='{.spec.template.spec.containers[?(@.name == "cluster-logging-operator")].image}')

I was suggesting more like below. 
A cluster administrator can follow the procedures on the troubleshooting page for OpenShift Logging issues. The following information can be gather for resolving OpenShift Logging related issues:        

* Checking the status of the Logging Operator. https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-cluster-status.html
* Checking the status of the Log store. https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-log-store-status.html
* Checking the OpenShift Logging alerts https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-alerts.html
* Collect information about your OpenShift Logging environment using oc adm must-gather https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-must-gather.html

**Comment by xenolinux** [comment]
> @xenolinux The new bullet point looks more Logging related than the general oc adm must-gather command.
> 
> The oc adm must-gather command is a bit different for Logging as it uses the cluster-logging-operator image for collecting OpenShift Logging related data. $ oc adm must-gather --image=$(oc -n openshift-logging get deployment.apps/cluster-logging-operator -o jsonpath='{.spec.template.spec.containers[?(@.name == "cluster-logging-operator")].image}')
> 
> I was suggesting more like below. A cluster administrator can follow the procedures on the troubleshooting page for OpenShift Logging issues. The following information can be gather for resolving OpenShift Logging related issues:
> 
>     * Checking the status of the Logging Operator. https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-cluster-status.html
> 
>     * Checking the status of the Log store. https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-log-store-status.html
> 
>     * Checking the OpenShift Logging alerts https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-alerts.html
> 
>     * Collect information about your OpenShift Logging environment using oc adm must-gather https://docs.openshift.com/container-platform/4.9/logging/troubleshooting/cluster-logging-must-gather.html

@IshwarKanse okay. Sounds like a good idea. I will add the logging section. Thanks for suggesting this!

**Comment by xenolinux** [comment]
Closing this PR and opening two new separate PRs. 

[1] https://github.com/openshift/openshift-docs/pull/39255

**Pull request closed without merging**